src/java/org/apache/cassandra/transport/CQLMessageHandler.java

```diff
@@ -178,7 +182,8 @@ protected boolean processOneContainedMessage(ShareableBytes bytes, Limit endpoin
 
         // max CQL message size defaults to 256mb, so should be safe to downcast
         int messageSize = Ints.checkedCast(header.bodySizeInBytes);
```

Comment:
Not related to your patch, but I think I forgot to throw an `@Override` annotation on `processOneContainedMessage()`